deploy-docker-image.sh

```diff
@@ -45,12 +45,22 @@ logout() {
   retry docker logout
 }
 
+deployQA() {
+  curl -X POST \
+    -F token=${DEPLOY_QA_TOKEN} \
+    -F ref=master \
+    -F variables[IMAGE_NAME]=inspirehep/hep \
```

Comment:
```suggestion
    -F variables[IMAGE_NAME]=inspirehep/next \
```

Reply:
there is also a call to make for `inspirehep/next-scrapyd` and `inspirehep/next-assets`, all using the same tag

Reply:
true :/